playwright.config.ts

```diff
@@ -18,9 +18,11 @@ export default defineConfig({
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 2 : undefined,
+  workers: process.env.CI ? 1 : undefined,
```

Comment:
@eth3lbert is there a reason why we're overriding `workers` on CI? did you notice any issues with it running in parallel? it appeared to work fine for me locally at least.

Reply:
I think we don't have enough CPU capacity to run both `QUnit` and `E2E` tests simultaneously.  And yeah, it's indeed safe to run them in parallel. I usually do that locally too :D